src/components/Form.svelte

```diff
@@ -116,14 +105,43 @@
         containerClass="col-span-3"
         options={['None', 'SASL_SSL', 'SASL_PLAINTEXT']}
       />
-      {#if $data.authMethod === 'SASL_PLAINTEXT'}
+      {#if $data.authMethod === 'None'}
+        <CheckboxField name="securedWithSSL" label="Secured with SSL" />
+        {#if $data.securedWithSSL}
+          <CheckboxField name="selfSignedCA" label="Self-signed certificate" />
+          {#if $data.selfSignedCA}
+            <SslForm errors={$errors.selfSignedCASsl} namePrefix="selfSignedCASsl" />
+          {/if}
+        {/if}
+      {:else if $data.authMethod === 'SASL_PLAINTEXT'}
+        <TextField
+          name="saslJaasConfig"
+          label="sasl.jaas.config"
+          errors={$errors.saslJaasConfig}
+          required
+        />
+      {:else if $data.authMethod === 'SASL_SSL'}
+        <CheckboxField name="securedWithSSL" label="Secured with SSL" />
```

Comment:
```suggestion
      {:else if $data.authMethod === 'SASL_SSL'}
        
```

$data.authMethod === 'SASL_SSL' means that securedWithSSL checked by default, and user have no ability to uncheck it.

Reply:
done